docs/operate/reference/viam-micro-server/manage-micro.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Manage Micro-RDK"
+linkTitle: "Manage Micro-RDK"
+weight: 100
+no_list: true
+type: docs
+tags: ["modular resources", "components", "services", "registry"]
+description: "Manage your Micro-RDK-based firmware."
+date: "2025-04-18"
+draft: true
```

Comment:
Confirming this page is currently hidden because of this flag?

Reply:
Correct. I will just delete it since it won't be reviewed properly for now.